docs/i18n-docs.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Automation tools for i18n and translate
+authors:
+- "@chouchongYHMing" # Authors' github accounts here.
+reviewers:
+- TBD
+approvers:
+- "@warjiang"
+
+creation-date: 2024-08-19
+
+---
+
+# Automation tools for i18n and translate
+
+## Summary
+
+This is an out-of-the-box automated i18n tool designed for the Karmada dashboard, developed primarily from a Chinese environment perspective. It implements the following features:
+
+1. Automatically configures i18n key-value pairs in the form of {key:value}.
```

Comment:
What's that mean, configure i18n key-value pair means extract chinese text and generate i18n kv pair.